### ISSUE TITLE
1.11.4-trail Build CPU-P9 Ray Failure

### ISSUE THREAD

**Issue by saisrivatsa** [open-ce/open-ce#1262]
Testing phase got failed .

```
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
+ python -c 'import ray; ray.init(ignore_reinit_error=True, num_cpus=4)'
2024-10-08 12:17:01,185 INFO worker.py:1783 -- Started a local Ray instance.
[2024-10-08 12:17:01,614 E 64481 64481] core_worker.cc:251: Failed to register worker to Raylet: IOError: [RayletClient] Unable to register worker with raylet. No such file or directory worker_id=01000000ffffffffffffffffffffffffffffffffffffffffffffffff
Tests failed for ray-core-2.35.0-py311h6d89b05_1.conda - moving package to <mydir>/broken
WARNING:conda_build.build:Tests failed for ray-core-2.35.0-py311h6d89b05_1.conda - moving package to <mydir>/broken
export PREFIX=<mydir>/ray-packages_1728387876552/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place
export SRC_DIR=<mydir>/ray-packages_1728387876552/test_tmp
import: 'ray'
import: 'ray._raylet'
import: 'ray.actor'
import: 'ray.runtime_context'
import: 'ray._private.state'
import: 'ray._private.worker'
import: 'ray'
import: 'ray._private.state'
import: 'ray._private.worker'
import: 'ray._raylet'
import: 'ray.actor'
import: 'ray.runtime_context'
TESTS FAILED: ray-core-2.35.0-py311h6d89b05_1.conda
```

**Comment by cdeepali** [comment]
This is a test stage failure with ray-core package. 

This error was not occurring when all the ray packages were installed. But was happening when ray-core alone was used. 

raylet logs at `/tmp/ray/session_latest/logs/ `showed:
```
[2024-10-10 14:15:46,732 E 944505 944552] (raylet) file_system_monitor.cc:111: /tmp/ray/session_2024-10-10_14-15-45_815771_944180 is over 95% full, available space: 7548821504; capacity: 294146834432. Object creation will fail if spilling is required.
Traceback (most recent call last):
  File "/opt/conda/envs/raycore/lib/python3.11/site-packages/ray/_private/runtime_env/agent/main.py", line 30, in <module>
    from aiohttp import web  # noqa: E402
    ^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'aiohttp'
[2024-10-10 14:15:47,163 E 944505 944582] (raylet) agent_manager.cc:83: The raylet exited immediately because one Ray agent failed, agent_name = runtime_env_agent.
The raylet fate shares with the agent. This can happen because
- The version of `grpcio` doesn't follow Ray's requirement. Agent can segfault with the incorrect `grpcio` version. Check the grpcio version `pip freeze | grep grpcio`.
- The agent failed to start because of unexpected error or port conflict. Read the log `cat /tmp/ray/session_latest/logs/{dashboard_agent|runtime_env_agent}.log`. You can find the log file structure here https://docs.ray.io/en/master/ray-observability/user-guides/configure-logging.html#logging-directory-structure.
- The agent is killed by the OS (e.g., out of memory).
```

Fixed by adding _aiohttp_ in the run section of ray-core package.

**Comment by cdeepali** [comment]
https://github.com/open-ce/ray-packages-feedstock/pull/23
